x/crisis/keeper.go

```diff
@@ -39,3 +39,12 @@ func (k *Keeper) RegisterRoute(moduleName, route string, invar sdk.Invariant) {
 func (k Keeper) Routes() []InvarRoute {
 	return k.routes
 }
+
+// return all the invariants
+func (k Keeper) Invariants() []sdk.Invariant {
```

Comment:
Can we have a unit test for just this function?

Reply:
I gracefully decline, I really _can't_ see how this is useful whatsoever, I'll add a `// DONTCOVER` though